src/projectDB/projectdatabase.cpp

```diff
@@ -186,7 +186,12 @@ int ProjectDatabase::saveGroupAndPeaks(PeakGroup* group,
                      , :fragmentation_spearman_rank_corr   \
                      , :fragmentation_tic_matched          \
                      , :fragmentation_num_matches          \
-                     , :sample_ids                         )");
+                     , :sample_ids                         \
```

Comment:
If user loads an emDB file from an older version of El-MAVEN, the slice will be zero for every saved group. As discussed offline, let's disable the Tolerance Lock for groups with zero slice.

Reply:
Will make the change.